_develop/connect/overview.md

```diff
@@ -8,22 +8,48 @@ permalink: /:collection/:path.html
 * TOC
 {:toc}
 
-Blockstack Connect is a JavaScript library for integrating your application with Stacks v2. With Connect, you get some big benefits:
+Blockstack Connect is a JavaScript library for integrating Blockstack authentication and smart contracts into your app.
 
-- Pre-built UI to introduce your users to Blockstack, before they authenticate
-- Integration with our new authenticator
-- Ability to interact with the Stacks 2.0 blockchain, with support for smart contracts and other transactions.
+The library empowers you to:
 
-## Demo
+- Register new users with a pre-built onboarding flow that quickly educates them as to the privacy benefits of using your app with Blockstack and provisions a "Secret Key" that secures their identity and data against the Stacks blockchain.
+- Authenticate users when they return to your app using that same Secret Key.
+- Prompt users to sign transactions with smart contracts as written in Clarity and published to the Stacks blockchain.
 
-To get a test feel for the user experience of using Connect, you can use [Banter](https://banter.pub).
+## See it in action
 
-## How does this compare to `blockstack.js`?
+Here's what it looks like to register new users into an app with Blockstack Connect:
+
+**insert video**
+
+As you can see, you simply need to add a "Get started" option to your app homepage and the library will display an introductory modal in the context of your app that explains the privacy benefits of using Blockstack and prepares users to receive a Secret Key instead of entering a traditional email and password.
+
+The modal then triggers a popup in which Blockstack PBC's authenticator generates a 12-word mnemonic Secret Key and instructs the user to save it securely. The authenticator subsequently requests a username from the user, which will be used to identify them in your app and manage their various accounts, before routing them back to your app to start using it.
```

Comment:
The first reference of a "Blockstack authenticator" - what is it? How's it related to Connect? Might need to set the context here

Reply:
Will do 👍